terraform/gcp.tf

```diff
@@ -110,16 +109,17 @@ resource "google_kms_crypto_key_iam_member" "vault-init" {
 
 # Get latest cluster version
 data "google_container_engine_versions" "versions" {
-  zone = "${var.zone}"
+  project = "${google_project.vault.project_id}"
+  region  = "${var.region}"
 }
 
 # Create the GKE cluster
 resource "google_container_cluster" "vault" {
   name    = "vault"
   project = "${google_project.vault.project_id}"
-  zone    = "${var.zone}"
+  region  = "${var.region}"
 
-  initial_node_count = "${var.num_vault_servers}"
+  initial_node_count = "${var.num_nodes_per_zone}"
```

Comment:
I'm not sure I understand this change? This will create 1 node per zone (which is usually 3-4 per region), but then there's an anti-affinity clause that prefers (but in practice refuses) to schedule more than one Vault pod on a single node (see vault.yaml). If the user chooses a `num_vault_servers` that's greater than the number of zones that the region has, the service will never start